src/Symfony/Component/HttpKernel/HttpCache/HttpCache.php

```diff
@@ -153,16 +153,10 @@ public function getKernel()
     /**
      * Gets the Surrogate instance.
      *
-     * @throws \LogicException
-     *
      * @return SurrogateInterface A Surrogate instance
      */
     public function getSurrogate()
     {
-        if (!$this->surrogate instanceof Esi) {
-            throw new \LogicException('This instance of HttpCache was not set up to use ESI as surrogate handler. You must overwrite and use createSurrogate');
```

Comment:
What is `createSurrogate`? I cannot find such definition. this exception still doesn't make sense to me in 2.7
Ref. https://github.com/symfony/symfony/pull/15306#discussion_r35737940